content/for-delivery-engineers/tutorials/preparing-env-for-pipeline-as-code/prepare-env.md

```diff
@@ -0,0 +1,115 @@
+# Setting Up Service Account, SCC, Role, and RoleBinding for Tekton Pipelines
+
+In this tutorial, we will guide you through the process of setting up a service account, a Security Context Constraint (SCC), a role, and a role binding for Tekton Pipelines. These steps are essential for managing your pipeline's security and access within your Kubernetes/OpenShift cluster.
+
+## Prerequisites
+
+- Access to a Kubernetes or OpenShift cluster.
```

Comment:
What kind of access? Customer delivery engineers can't be clusteradmins; maybe SCA? but then you should validate as SCA and not kubeadmin and document as SCA and not kubeadmin